### ISSUE TITLE
Multiple Tags to a location

### ISSUE THREAD

**Issue by dummifiedme** [esm7/obsidian-map-view#32]
Sometimes same location is required to be marked with multiple tags so that when we filter it out, it would show in each tag filter :)

I tried various ways, nothing seems to work.
```md
[map](geo: 21,12) tag:tag1 tag:tag2 

[map](geo: 21,12) tag:tag1,tag2 

[map](geo: 21,12) tags:tag1,tag2

[map](geo: 21,12) tag:[tag1.tag2]

[map](geo: 21,12) tags:[tag1.tag2]
```

If there is this feature already, please direct me :)
else, please consider this enhancement.

**Comment by dummifiedme** [comment]
At the time of posting this issue, I wasn't aware that it considers other tags in the file as well. 
I was wondering it can create issues with filtering.

So, if possible, we can have some kind of an optional inline flag, that makes it exclusive inline tag

e.g.

```
[map](geo: 21,12) & tag:[tag1,tag2]

```

Just an example.

**Comment by esm7** [comment]
The basic idea was that the file tags (the regular Obsidian `#tag` ones) are the base ones you want to use for all locations in the note, and on top of that you can add specific ones for specific locations.
Just like you were not aware of this, I'm concerned that I make more custom syntax with more options, this will confuse users even further :sweat_smile:

**Comment by dummifiedme** [comment]
Yes, sure. Maybe then just having multi tag support would be great

**Comment by esm7** [comment]
Just released 1.1.0 which adds support for this.
